src/ServiceBuilder.php

```diff
@@ -92,7 +92,7 @@ public function __construct(array $config = [])
      * ```
      * use Google\Cloud\ServiceBuilder;
      *
-     * // Create a storage client using application default credentials.
+     * // Create a Storage client using application default credentials.
```

Comment:
Having the comment directly above the service builder instantiation seems to indicate it describes that line specifically. Maybe we should add some sort of note at the top of each page:

> All examples use the application default credentials